go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/mgutz/ansi v0.0.0-20200706080929-d51e80ef957d // indirect
 	github.com/nsf/termbox-go v0.0.0-20160718140619-0723e7c3d0a3
 	github.com/pkg/errors v0.9.1
-	github.com/smartystreets/assertions v0.0.0-20160205033931-287b4346dc4e // indirect
+	github.com/smartystreets/assertions v0.0.0-20190401211740-f487f9de1cd3 // indirect
```

Comment:
Had to update this for some assertions related to `time.Time` to work.

Reply:
Sure, that should work. The way I just had it (with `assertions` being 3 years newer than `goconvey`) is the way mtc has it set up.

I'll update to the most recent version of `goconvey` (1.6.4), which looks to actually get rid of the pinned indirect dependencies in `go.mod`.